packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -418,6 +428,21 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
     this.handlers.onRemoveParticipant(userId);
   }
 
+  /* @conditional-compile-remove(PSTN-calls) */
+  public async addParticipant(participant: CommunicationIdentifier, options?: AddPhoneNumberOptions): Promise<void> {
+    this.handlers.onAddParticipant(participant, options);
+  }
+
+  /* @conditional-compile-remove(PSTN-calls) */
+  public async holdCall(): Promise<void> {
+    this.handlers.onToggleHold();
```

Comment:
note: this can't blindly call `onToggleHold`, it will need to check if the call handlers.holdCall() or check if the call isn't already on hold